src/apps/rDSN.monitor/app_package/MonitorApp.py

```diff
@@ -317,43 +505,194 @@ def get(self):
+        self.SendJson(clusterList)
+        
+
+class ApiServicelistHandler(BaseHandler):
+    def post(self):
+        serviceList = []
+        instance_name = 'testrun'
+        package_name = 'rdsn'
+        cluster_name = 'cluster1'
+        state = 'deployed'
+        error = ''
+
+        #req = {"package_id":""}
+        #self.response.write(Native.dsn_cli_run('service_list ' + json.dumps(req)))
+        serviceList.append({'instance_name':instance_name,'package_name':package_name,'cluster_name':cluster_name,'state':state,'error':error})
+        self.SendJson(serviceList)
 
 def start_http_server(portNum):  
-    static_app = webob.static.DirectoryApp(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))+"/static")
+    static_app = webob.static.DirectoryApp(GetMonitorDirPath() + "/static")
     web_app = webapp2.WSGIApplication([
-    ('/', mainHandler),
-    ('/main.html', mainHandler),
-    ('/table.html', tableHandler),
-    ('/perValue1.html', perValue1Handler),
-    ('/perValue2.html', perValue2Handler),
-    ('/perValue3.html', perValue3Handler),
-    ('/perValue5.html', perValue5Handler),
-    ('/consoleCli.html', consoleCliHandler),
-    ('/execCli.html', execCliHandler),
-    ('/consoleBash.html', consoleBashHandler),
-    ('/execBash.html', execBashHandler),
-    ('/editor.html', editorHandler),
-    ('/configure.html', configureHandler),
-    ('/selectDisplay.html', selectDisplayHandler),
-    ('/clusterinfo.html', clusterinfoHandler),
-
-    ('/perValue2', perValue2QueryHandler),
-    ('/psutil', psutilQueryHandler),
-    ('/clusterinfo', clusterQueryHandler),
- 
+    ('/', PageMainHandler),
+    ('/main.html', PageMainHandler),
+    ('/table.html', PageTableHandler),
+    ('/sample.html', PageSampleHandler),
+    ('/value.html', PageValueHandler),
+    ('/bar.html', PageBarHandler),
+    ('/queue.html', PageQueueHandler),
+    ('/cli.html', PageCliHandler),
+    ('/bash.html', PageBashHandler),
+    ('/editor.html', PageEditorHandler),
+    ('/configure.html', PageConfigureHandler),
+    ('/fileview.html', PageFileViewHandler),
+    ('/analyzer.html', PageAnalyzerHandler),
+    ('/view.html', PageViewHandler),
+    ('/store.html', PageStoreHandler),
+    ('/service.html', PageServiceHandler),
+
+    ('/api/cli', ApiCliHandler),
+    ('/api/bash', ApiBashHandler),
+    ('/api/value', ApiValueHandler),
+    ('/api/psutil', ApiPsutilHandler),
+    ('/api/remoteCounterSample', ApiRemoteCounterSampleHandler),
+    ('/api/remoteCounterCalc', ApiRemoteCounterCalcHandler),
+    ('/api/view/save', ApiSaveViewHandler),
+    ('/api/view/load', ApiLoadViewHandler),
+    ('/api/view/del', ApiDelViewHandler),
+    ('/api/pack/load', ApiLoadPackHandler),
+    ('/api/pack/del', ApiDelPackHandler),
+    ('/api/pack/deploy', ApiDeployPackHandler),
+    ('/api/pack/undeploy', ApiUndeployPackHandler),
+    ('/api/clusterlist', ApiClusterlistHandler),
```

Comment:
why specific handlers for those that can be purely handled by cli ? Isn't this the same to /api/cli/clusterlist?